packages/material-ui-cssutils/README.md

```diff
@@ -0,0 +1,19 @@
+# @material-ui/cssutils
+
+You can leverage our CSS helper functions that integrate with the rest of Material-UI.
+
+## Installation
+
+Install the package in your project directory with:
+
+```sh
+// with npm
+npm install @material-ui/cssutils
+
+// with yarn
+yarn add @material-ui/cssutils
+```
+
+## Documentation
+
+So far, we have implemented a [helper function](https://material-ui.com/customization/themes#responsive-fonts) for generating responsive fonts.
```

Comment:
This needs to document purpose and utilisation of the utils in `cssUtils`, rather than the theme helper.

Reply:
I was suggesting that it's documented here, in this section README. The docs will be available through npmjs.com, or when browsing the repo. Other than a link, I don't think we need to document it in the main docs site.

Reply:
For this kind of helpers, I think that we should automatically generate the documentation, from the code comments. It's the strategy used by lodash for instance. Now, it's more important to release it, so we can work on it afterward.

Reply:
That would be useful for `colorManipulator`, utils as I have already added comments documenting the API.

Reply:
>we have to document the options, what's going on, how it works.

>(...) For this kind of helpers, I think that we should automatically generate the documentation, from the code comments

I initially saw examples of JSDoc in the codebase and thought you used that, so I had added JSDoc annotations (I was going to handle the doc file generation bit in a separate MR):

* https://github.com/mui-org/material-ui/blob/325ada47193920d5a6e52711e93853f450f5a7e2/packages/material-ui/src/styles/responsiveTypography.js#L10-L22
* https://github.com/mui-org/material-ui/blob/325ada47193920d5a6e52711e93853f450f5a7e2/packages/material-ui-css-utils/src/responsiveProperty.js#L3-L14

However further reviewing I see that

1. you seem to define APIs in the markdown docs ([example](https://next.material-ui.com/customization/themes/#createmuitheme-options-theme))
2. I'm not finding the use of the few JSDoc annotations that I found

I presume JSDoc adoption is not forthcoming? I can just move to the `theme.md` (for `responsiveTypography` in `material-ui/core`) and just let me know which mechanism to adopt for generating docs, and I can use that in a separate MR for functions within css-utils?

>how do we organize the documentation? We start to have too much content.

>(...) I suspect the @material-ui prefix will always be a barrier to that [communicate the fact that it has nothing to do with Material Design]

I guess these are related, you could have this package hosted separately to material-ui (different branding), reducing the amount of docs here. But probably early to do this.